app/models/reports/facility_state.rb

```diff
@@ -8,5 +8,9 @@ def self.for_region(region_or_source)
       region = region_or_source.region
       where("#{region.region_type}_region_id" => region.id)
     end
+
+    def period_month_date
+      Period.month(month_date)
+    end
```

Comment:
how about just `period`? `facility_state.period` seems clear to me.

Reply:
done